packages/babel-helper-create-class-features-plugin/src/index.js

```diff
@@ -120,6 +120,18 @@ export function createClassFeaturePlugin({
           }
 
           if (!isDecorated) isDecorated = hasOwnDecorators(path.node);
+
+          if (path.isStaticBlock()) {
```

Comment:
```suggestion
          if (path.isStaticBlock?.()) {
```

Otherwise this will fail with older `@babel/traverse` versions